templates_eventbridge/EventBridge.yaml

```diff
@@ -0,0 +1,60 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: EventBridgeRole and EventBridgeRole for falco cloud
+Parameters:
+  SysdigName:
+    Type: String
+    Default: "sysdig"
```

Comment:
Again, should this role name be consistent?

Reply:
Its renamed to SysdigEBName